src/scripts/main.js

```diff
@@ -1,3 +1,209 @@
+  rows.forEach((row) => tableBody.appendChild(row));
+});
+
+salarySort.addEventListener('click', (e) => {
+  e.stopPropagation();
+
+  const rows = Array.from(document.querySelectorAll('tbody > tr'));
+
+  rows.sort((a, b) => {
+    const salaryA = +a
+      .querySelector('td:nth-child(5)')
+      .textContent.slice(1)
+      .split(',')
+      .join('');
+    const salaryB = +b
+      .querySelector('td:nth-child(5)')
+      .textContent.slice(1)
+      .split(',')
+      .join('');
+
+    return isAscendSalary ? salaryA - salaryB : salaryB - salaryA;
+  });
+  isAscendSalary = !isAscendSalary;
+  tableBody.innerHTML = '';
+  rows.forEach((row) => tableBody.appendChild(row));
+});
+
+rowsForActive.forEach((row) => {
+  row.addEventListener('click', (e) => {
+    e.stopPropagation();
+
+    rowsForActive.forEach((r) => {
+      r.classList.remove('active');
+    });
+    row.classList.add('active');
+  });
+});
+
+body.insertAdjacentHTML(
+  'beforeend',
+  `<form class="new-employee-form">
+  <label>Name: <input name="name" type="text" data-qa="name" required></label>
+  <label>Position: <input name="position" type="text" data-qa="position" required></label>
+  <label>Office:
+    <select name="office" type="text" data-qa="office" required>
+      <option value="Tokyo">Tokyo</option>
+      <option value="Singapore">Singapore</option>
+      <option value="London">London</option>
+      <option value="New York">New York</option>
+      <option value="Edinburgh">Edinburgh</option>
+      <option value="San Francisco">San Francisco</option>
+    </select>
+  </label>
+  <label>Age: <input name="age" type="number" data-qa="age" required></label>
+  <label>Salary: <input name="salary" type="number" data-qa="salary" required></label>
+  <button>Save to table</button>
+  </form>
+  `,
+);
+
+const button = document.querySelector('button');
+
+button.addEventListener('click', (e) => {
+  e.preventDefault();
+  e.stopPropagation();
+
+  const userName = document.querySelector('[data-qa="name"]').value;
+  const userPosition = document.querySelector('[data-qa="position"]').value;
+  const userOffice = document.querySelector('[data-qa="office"]').value;
+  const userAge = document.querySelector('[data-qa="age"]').value;
+  const salaryInputValue = document.querySelector('[data-qa="salary"]').value;
+  const userSalary = `$${Number(salaryInputValue).toLocaleString('en-US')}`;
+
+  if (isValidInputs(userName, userAge)) {
+    tableBody.insertAdjacentHTML(
+      'beforeend',
+      `<tr>
+        <td>${userName}</td>
+        <td>${userPosition}</td>
+        <td>${userOffice}</td>
+        <td>${userAge}</td>
+        <td>${userSalary}</td>
+      </tr>`,
+    );
+
+    notification.classList.add('success');
+    notificationTitle.textContent = 'Congrats!!!';
+    notificationMessage.textContent = 'Succesfully added new employee!!!';
+    body.appendChild(notification);
+
+    setTimeout(() => {
+      notification.style.visibility = 'hidden';
```

Comment:
The notification's visibility is set to 'hidden', but it is never reset to 'visible'. This means that once the notification is hidden, it will not appear again unless the visibility is manually reset. Consider setting the visibility back to 'visible' when showing the notification.